pkg/apis/deployment/secrets.go

```diff
@@ -108,6 +101,18 @@ func (deploy *Deployment) createOrUpdateSecrets(registration *radixv1.RadixRegis
 	return nil
 }
 
+func (deploy *Deployment) getBlobFuseCredsSecrets(ns, componentName string) (string, []byte, []byte) {
+	secretName := defaults.GetBlobFuseCredsSecretName(componentName)
+	accountKey := []byte(secretDefaultData)
+	accountName := []byte(secretDefaultData)
```

Comment:
Why are the slices initialized a dummy value instead of an empty slice?
```suggestion
	accountName := []byte{}
```